frontend/src/mock_data/contract_template.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Why are you adding all this singleton data? We have pluralized versions of all these files already in the repo.

Reply:
this was the old original JSON file... will be removed in the next commit

Reply:
that's weird because all those singleton data were added when I first started this PR... not from my recent commit

Reply:
Maybe you need to merge upstream again?

Reply:
I think it's because I didn't create a new branch from master. Instead, I reopened the previously merged PR... Should I close this one and create a new PR?

Reply:
I think it's the renaming issue on GIT again... I suppose the mock data JSON files have been renamed to their pluralized forms. Then git added them as untracked files, instead of replacing the original singleton files